src/classes/Foo5.cls

```diff
@@ -0,0 +1,15 @@
+public class Foo5 {
```

Comment:
Class `Foo5` does not enforce sharing correctly. Please change to `Inherited Sharing`.